tests/test_meshtastic_utils_health.py

```
@@ -54,28 +54,21 @@ async def test_check_connection_ble_skips_health_checks(reset_meshtastic_globals
         await check_connection()
 
-    mock_logger.info.assert_any_call(
-        "BLE connection uses real-time disconnection detection - health checks disabled"
+    mock_logger.debug.assert_any_call(
+        "BLE connection uses real-time disconnection detection; periodic health checks disabled"
     )
 
 
 @pytest.mark.asyncio
-async def test_check_connection_metadata_fallback_succeeds(reset_meshtastic_globals):
+async def test_check_connection_node_info_probe_succeeds(reset_meshtastic_globals):
```

Ruff (0.15.2)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)